### ISSUE TITLE
Behavior of dropping a container onto an automation is not as expected

### ISSUE THREAD

**Issue by salkin-mada** [ossia/score#553]
Say a container consists of 5 floats. 
* When dropping the container onto an empty automation it does not make 4 new processes in the automation and adds all 5 floats. It just connects the first float to the address of the one process in automation.

**Comment by bltzr** [comment]
another way to phrase it would be: create one automation for each address when dropped on an empty automation (or, probably, more generally, on a constraint), right ?

**Comment by jcelerier** [comment]
actually that's what happens if you drop them on the constraint.

**Comment by salkin-mada** [comment]
Yes when dropped on a empty constraint it happens. But not when dropped on an automation.
@bltzr yes. I would suggest this behavior: __create one automation for each address when dropping a container on an empty automation.__
